libexec/cli/list.exec

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+# 
+# Copyright (c) 2017, SingularityWare, LLC. All rights reserved.
+#
+# This software is licensed under a 3-clause BSD license.  Please
+# consult LICENSE file distributed with the sources of this project regarding
```

Comment:
If you agree with my suggestion, this script would be called `ps.exec`. If we anticipate different functions for daemons,  (or should we call them instances?) then we may want to do something like `instance-ps.exec` or `daemon-ps.exec`